packages/frontend-2/components/header/NavLink.vue

```diff
@@ -31,3 +31,11 @@ defineProps({
   }
 })
 </script>
+
+<style>
+.active-class .maybe-truncate {
+  overflow: hidden;
```

Comment:
You can leave this as is, but in the future you can use Tailwind's `@apply XXX` directives, to construct new classes out of existing Tailwind classes.

```
.foo {
  @apply truncate font-bold text-foreground
}
```